### PR TITLE
save password

### PR DESCRIPTION
**Pull request by AlselwiAisha**
Add dotenv-rails gem this gem is particularly useful when deploying applications to different environments, ensuring that sensitive information remains secure and that configurations can be easily managed across various deployment scenarios.

I used .env to add DEVISE_JWT_SECRET_KEY in.

update database delete column password from user table

### CODE REVIEW

**Review by Chimoneg27 — CHANGES_REQUESTED** [review]
### Required changes♻️

- Great work so far but there are some issues me and @sadaf-Daneshgar are having when we try to login
- May you please show us how you login after lunch

**Comment by AlselwiAisha** [comment]
Please try to clone it again

File: db/migrate/20240118134238_create_users.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@ class CreateUsers < ActiveRecord::Migration[7.1]
   def change
     create_table :users do |t|
       t.string :name
-      t.string :password
+      
       t.timestamps
```
Comment (Arch-Noize) [inline db/migrate/20240118134238_create_users.rb:6]: If a user is to sign in... how can they do so without a password? In our [frontend](https://github.com/leloufadel/front-end-capstone/pull/7) we still call and use passwords to make our calls and fill our user details. So every time we create a user, we still give it a "password" even though the user in our database doesn't actually have a password anymore thanks to this change.

Why remove it?
image

**Review by Arch-Noize — COMMENTED** [review]
# A couple questions

I can agree the code seems to be working fine without errors. There's something I don't quite understand, please look into this

File: db/migrate/20240118134238_create_users.rb (hunk 1 above)
Comment (Chimoneg27) [reply db/migrate/20240118134238_create_users.rb:6]: Good question. I'd didn't notice. @AlselwiAisha can explain it to us tomorrow. I think she's not available right now

**Review by Arch-Noize — APPROVED** [review]
# Approved! :green_circle: :green_circle: 

Well done on implementing loging in. There's nothing left to do but to merge!

**Review by Chimoneg27 — APPROVED** [review]

**Pull request merged**